### PR TITLE
Hotfix/refactor global nav

### PR DESCRIPTION
**Pull request by NicolaSaunders**

### CODE REVIEW

**Comment by vivienlacourba** [comment]
Fixes  #234

**Comment by vivienlacourba** [comment]
Fixes https://github.com/w3c/w3c-website/issues/234

File: assets-src/js/main.js [hunk 1]
```diff
@@ -9,14 +11,25 @@ import {flashes} from "./main/flashes";
...
 	disclosureWidget();
 	formErrorSummary();
 	responsiveTables();
 	flashes();
-};
+
+	/* Create a navDoubleLevel object and initiate double-level navigation for a <ul> with the correct data-component attribute */
+	const navDoubleIntro = document.querySelector('ul[data-component="nav-double-intro"]');
+
+	if (exists(navDoubleIntro)) {
```
Comment (deniak) [inline assets-src/js/main.js:24]: I don't get why we need a dedicated function to check if the element exists.
Isn't checking if `navDoubleIntro` is null enough?

Same comment applies each time that function is used in the other files.

**Review by deniak — CHANGES_REQUESTED** [review]

File: assets-src/js/main.js (hunk 1 above)
Comment (NicolaSaunders) [reply assets-src/js/main.js]: Thanks. I've double checked and we can indeed simplify the if statements and remove the that function.

I've just [pushed another commit](https://github.com/w3c/w3c-website-templates-bundle/pull/103/commits/0f645f32f031894cde1a8d45b4a773ecfbc4b39b) for this (and some other bits of tidying in the JS files):

**Review by deniak — APPROVED** [review]

**Pull request merged**